application/controllers/artifacts_controller.py

```diff
@@ -68,6 +68,20 @@ def update(self, params, object_id):
         except NotFound:
             return {"error": "not found"}, 404
 
+    @use_args(artifacts_validator.update_many_args())
+    def update_many(self, params):
+        """ Logic for updating multiple artifacts at once """
+
+        for update_params in params["data"]:
```

Comment:
rename they key to `images` and the `update_params` to `image` imo. That would be a lot more descriptive.

Reply:
Hmm but why image and not artifacts?